app/components/e-heading.md

```diff
@@ -0,0 +1,51 @@
+### Headings
+
+```
+<div>
+  <e-heading tagName="h1" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h2" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h3" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h4" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h5" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <a href="#"><e-heading tagName="h6" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading></a>
```

Comment:
The Link adds borders around element. Also link should be defined inside `h` tag because `h` is block and `a` is inline. It's ok to overwrite the link style inside `e-heading` in this case.

- [ ] Remove link borders
- [ ] Move link inside header